openlibrary/templates/books/edition-sort.html

```diff
@@ -85,5 +85,13 @@
         <ul class="read">
           <li class="read-option">$:macros.LoanStatus(book)</li>
         </ul>
+        $if oclc_numbers or isbn:
+          <ul class="world-cat-link">
```

Comment:
a different class should be used here, using the BEM syntax since this is a new element.
suggested: editions-table__world-cat-link

Reply:
Done